package.json

```diff
@@ -12,9 +12,10 @@
     "validate-icons": "node scripts/validate-meta.js",
     "generate-meta": "node scripts/generate-meta.js",
     "compile-styles": "sass --style=expanded ./src/index.scss dist/index.css",
+    "compile-mixins": "cat ./src/mixins/*.scss > ./dist/mixins.scss",
```

Comment:
sass always compile scss to css and if the mixins are not used they will not be part of the compiled version. we do not wan't to compile our mixins, just to have them in one exported file

(did not add docs info to readme it will be part of the docs phase)